elide-annotations/src/main/java/com/yahoo/elide/security/checks/prefab/Common.java

```diff
@@ -34,4 +33,17 @@ public boolean ok(T record, RequestScope requestScope, Optional<ChangeSpec> chan
             return false;
         }
     }
+
+    /**
+     * A check that enables Only teh removal of an object from a relationship.
```

Comment:
A few minor types.
`Only` -> `only`
`teh` -> `the`

That said, for some reason this comment is unclear to me. Can we maybe reword it?